CMakeLists.txt

```diff
@@ -498,6 +498,7 @@ if(BUSTED_PRG)
         -DTEST_DIR=${CMAKE_CURRENT_SOURCE_DIR}/test
         -DBUILD_DIR=${CMAKE_BINARY_DIR}
         -DTEST_TYPE=unit
+        -DCMAKE_SYSTEM_NAME=${CMAKE_SYSTEM_NAME}
```

Comment:
Do we really need to pass this into the "target CMake"? Isn't it just defined there by default as well?